hearts/hearts.py

```diff
@@ -204,7 +204,7 @@ def upkeep(self, player, card):
 
     def play_card(self, player, card):
         if self.is_player_turn(player):
-            if len(self.tricks) == 0 or len(self.tricks[-1]) == 4:
+            if len(self.tricks) == 0 or self.tricks[-1].size == 4:
```

Comment:
I see the issue.  Can we review the test_breaking_hearts() and merge it in if it looks ok?